book/website/reproducible-research/reviewing/reviewing-challenges.md

```diff
@@ -0,0 +1,122 @@
+
+A common outcome is that a pull request will be left in the queue for months or even years without being addressed.
+This is almost guaranteed to alienated potential members of the community who might otherwise make more contributions.
+
+It's crucial that reviews are tackled in a timely manner.
+For an open source project, it is the responsibility of the project maintainers, and a sign of good stewardship, if pull requests are reviewed in good time.
+
+Allowing stale pull requests to build up can cause drag on a project.
+
+If a project isn't ready to accept a pull request or doesn't have the capacity to review it, it is beneficial to make this clear, either by rejecting the pull request, or by engaging with the author to explain the situation.
+
+## The infinite review
+
+If a reviewer doesn't like a piece of code, it's not unusual for them to simply leave a review incomplete, rather than tackle the thorny issues it raises.
+This may be especially the case if a reviewer is having difficulty articulating what they dislike about a proposed change.
+
+Leaving pull requests half reviewed, from the side of the author or reviewer, can feel disrespectful to other parties who have worked on it.
+
+If there is any uncertainty about the benefits or purpose of a particular change, it is better to make this clear through a comment, emphasising that you're looking to better understand.
+
+If a change is ultimately not going to be merged, it is more respectful to reject explicitly than to leave the review incomplete.
+When rejecting a change it is essential that the reason is explained for the benefit of the author.
+
+Remember that not everything has to be resolved in a single pull request.
+If it looks like it's going to be hard to come to a decision on something, create a new issue to deal with it, rather than allow it to block the code merge.
+
+## Bad contributions are better than no contributions
+
+As a reviewer it may be necessary to accept the code in a form that you're not happy with.
+In this case, create an issue to fix parts of it that may be problematic in a separate issue.
+It can sometimes be quicker to fix an issue yourself rather than work through someone else to make the change.
+
+This is especially true early on in a project when a community is still being built up around it.
+Contributors who make poor submissions now, with help and encouragement, may turn into the driving force of a project in the future.
+Reviews should always be encouraging and especially in the early stages, it can be better to err on the side of accepting rather than alienate potential collaborators.
+
+## The weight of responsibility
+
+When performing a review it can be helpful to have an appreciation of who is ultimately responsible for the changes.
+Reviewers who feel overly responsible are liable to spend too much time and be too critical in their reviews.
```

Comment:
This is an interesting point.

It might also be worth linking back to the idea of a 'core' team of developers. Although in this section you say the author of the code is responsible, it seems likely that it is the maintainers who will catch the flak of any problems and have to organise a fix. The author may not even get notification that there was a problem.